projects/mercury/src/common/utils/httpUtils.js

```diff
@@ -51,6 +51,25 @@ export function handleHttpError(defaultMessage) {
     };
 }
 
+export function handleRemoteSourceHttpError(defaultMessage) {
```

Comment:
It would be useful to have a comment over the method explaining why it is different for external source